### PR TITLE
Send reports via Slack

### PR DESCRIPTION
**Pull request by vitorarins**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -23,6 +24,12 @@ var (
 	action   = kingpin.Flag("action", "Any of the following actions: audit.").Envar("ESTAFETTE_EXTENSION_ACTION").String()
 	level    = kingpin.Flag("level", "Level of security you want to check for. It can be: low, moderate, high or critical.").Default("low").OverrideDefaultFromEnvar("ESTAFETTE_EXTENSION_LEVEL").String()
 	devLevel = kingpin.Flag("dev-level", "Level of security you want to check for your Dev dependencies. It can be: low, moderate, high, critical or none.").Default("low").OverrideDefaultFromEnvar("ESTAFETTE_EXTENSION_DEV_LEVEL").String()
+
+	// slack flags
+	slackWebhookURL      = kingpin.Flag("slack-webhook-url", "A slack webhook url to allow sending messages.").Envar("ESTAFETTE_EXTENSION_SLACK_WEBHOOK_URL").String()
```
Comment (JorritSalverda) [inline main.go]: Since this needs no backwards compatibility, like the slack build status extension does you can drop this flag.

File: main.go [hunk 2]
```diff
@@ -46,72 +56,76 @@ func main() {
...
+			} else {
+				log.Println(err)
+			}
 		} else {
-			// TODO: send report via Slack
-			log.Println("Auditing passed, but you might still have vulnerabilities.")
+			log.Println("No vulnerabilities in your repository for now. Cheers!")
 		}
 	default:
 		log.Fatal("Set `action: <action>` on this step to audit.")
```
Comment (JorritSalverda) [inline main.go:115]: Perhaps repeat allowed values for `<action>`.

**Comment by JorritSalverda** [comment]
I'll configure the extension as trusted and inject the slack credentials.

File: main.go (hunk 2 above)
Comment (vitorarins) [reply main.go:115]: But action for now it can only be `audit`

**Review by JorritSalverda — APPROVED** [review]

**Pull request merged**